jobs/payment-jobs/tasks/common/enums.py

```diff
@@ -30,3 +30,9 @@ class PaymentDetailsGlStatus(Enum):
     INPRG = 'INPRG'
     RJCT = 'RJCT'  # Should have refundglerrormessage
     CMPLT = 'CMPLT'
+
+
+class OrgStatus(Enum):
```

Comment:
Not sure if we need this anymore

Reply:
Good catch, no longer needed and removed